docs/waivers.md

```diff
@@ -0,0 +1,54 @@
+# Waivers (Beta)
+
+Waivers is a mechanism to mark controls as "waived" for various reasons
+and to control the running and/or reporting of those controls. It
+takes a YAML input file that identifies which controls are waived,
+(optionally) whether they should be skipped from running, a description of why it
+is waived, and (optionally) an expiration date for the waiver.
+
+NOTE: This mechanism is currently in beta and is ultimately meant to be
+ingested and maintained by Chef Automate UI.
```

Comment:
I think Product can speak more to this, but the feature is certainly exposed in such a way that is not bound to Automate. I'm not sure the docs should limit it to that scope.

Reply:
In standup the hangup seemed to be around "ultimately" being interpreted as "the only way" when I meant it as a future looking thing. We have many features that were intended for specific uses/users but can be used by many.

I'm changing this to "intended" after talking to Mary.